gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannel.java

```diff
@@ -223,7 +225,12 @@ private Object doResumableUpload() throws IOException {
 
       private final long writeOffset;
       private final String uploadId;
+      private final TreeMap<Long, ByteString> dataChunkMap;
```

Comment:
Add a comment, "Maps from x to y ..." (i.e., what is the Long, what is the ByteString)?

Reply:
Docstring added.